src/t_zset.c

```diff
@@ -3927,21 +3936,20 @@ void zpopMinMaxCommand(client *c, int where) {
     }
 
     long count = 0; /* 0 for plain single pop. */
+    int reply_empty_with_zero_count = 0;
```

Comment:
```suggestion
    long count = -1; /* default of -1 mean a plain single pop. */
```

Reply:
I thought about it before, but in the LMPOP/ZMPOP commands, i use 0 for single pop.
If we change that, we need to touch a lot of code, even in BLOCK commands...

I can take a another look deeply tomorrow